terminus/geometry/arc.py

```diff
@@ -102,10 +170,14 @@ def includes_point(self, point, buffer=1e-7):
         if abs(center_point.norm() - self._radius) > buffer:
             return False
         angle_between_vectors = center_start.angle(center_point)
-        if angle_between_vectors == 0:
+        if abs(angle_between_vectors) < buffer:
+            # this solves some problems that come from approximate calculations
             return True
-        angular_delta = abs(angle_between_vectors) - abs(self._angular_length)
-        return (angle_between_vectors < 0) == (self._angular_length < 0) and angular_delta < buffer
+        if self._angular_length >= 0:
+            return 1e-5 >= self._clip_angle_to_360(angle_between_vectors) - self._angular_length
```

Comment:
Just curious: doesn't the buffer (`1e-7`) work here? It would be nice if all the tolerance could be parametrized